Sprint-3/implement/is-proper-fraction.js

```diff
@@ -32,3 +32,21 @@
 // target output: false
 // Explanation: The fraction 3/3 is not a proper fraction because the numerator is equal to the denominator. The function should return false.
 // These acceptance criteria cover a range of scenarios to ensure that the isProperFraction function handles both proper and improper fractions correctly and handles potential errors such as a zero denominator.
+
+function isProperFraction(numerator, denominator){
+    if (numerator < denominator) {
```

Comment:
Handle negative fractions by using the absolute value of the numerator to ensure proper fraction logic works consistently for both positive and negative numerators.

Reply:
Ok